src/SignalR/clients/csharp/Client/src/HttpConnectionFactory.cs

```diff
@@ -16,16 +18,23 @@ namespace Microsoft.AspNetCore.SignalR.Client
     /// </summary>
     public class HttpConnectionFactory : IConnectionFactory
```

Comment:
Make this internal. I want to make this properly public in the right assembly

Reply:
Why not move it to the right assembly now?

Reply:
I'm assuming you want this in the Http.Connections.Client assembly, right? The layering made no sense before, so I didn't want to move things around until I was sure I understood your vision. Especially since the move would technically be breaking.

Reply:
I didn't move this to Http.Connections.Client yet, because it needs to be accessible from HubConnectionBuilderHttpExtensions.